### PR TITLE
Readme updated for default template

### LINKED ISSUE #2289: Default workspace README needs info about TS
The Default workspace currently does not mention TS Scripts:
This line needs to be updated:
**The 'scripts' folder contains two example async/await scripts for deploying the 'Storage' contract.**
There are now 4 scripts - 2 js and 2 ts

### PR DESCRIPTION
**Pull request by Aniket-Engg**
fixes #2289

### CODE REVIEW

File: libs/remix-ws-templates/src/templates/remixDefault/README.txt [hunk 1]
```diff
@@ -2,25 +2,27 @@ REMIX DEFAULT WORKSPACE
...
-2. 'scripts': Holds two scripts to deploy a contract. It is explained below.
-3. 'tests': Contains one Solidity test file for 'Ballot' contract & one JS test file for 'Storage' contract
+2. 'scripts': Contains four typescript files to deploy a contract. It is explained below.
+3. 'tests': Contains one Solidity test file for 'Ballot' contract & one JS test file for 'Storage' contract.
 
 SCRIPTS
 
-The 'scripts' folder contains two example async/await scripts for deploying the 'Storage' contract.
-For the deployment of any other contract, 'contractName' and 'constructorArgs' should be updated (along with other code if required). 
+The 'scripts' folder have four typescript files which help to deploy the 'Storage' contract using 'web3.js' and 'ethers.js' libraries.
```
Comment (ryestew) [inline libs/remix-ws-templates/src/templates/remixDefault/README.txt:16]: The 'scripts' folder has four

File: libs/remix-ws-templates/src/templates/remixDefault/README.txt [hunk 2]
```diff
@@ -2,25 +2,27 @@ REMIX DEFAULT WORKSPACE
...
 
-Also, there is a script containing some unit tests for Storage contract inside tests directory.
+For the deployment of any other contract, just update contract name from 'Storage' to desired contract and provide constructor arguments accordingly
```
Comment (ryestew) [inline libs/remix-ws-templates/src/templates/remixDefault/README.txt:18]: For the deployment of any other contract, just update the contract's name from 'Storage' to the desired contract and provide constructor arguments accordingly

File: libs/remix-ws-templates/src/templates/remixDefault/README.txt [hunk 3]
```diff
@@ -2,25 +2,27 @@ REMIX DEFAULT WORKSPACE
...
 
-The 'scripts' folder contains two example async/await scripts for deploying the 'Storage' contract.
-For the deployment of any other contract, 'contractName' and 'constructorArgs' should be updated (along with other code if required). 
+The 'scripts' folder have four typescript files which help to deploy the 'Storage' contract using 'web3.js' and 'ethers.js' libraries.
 
-Also, there is a script containing some unit tests for Storage contract inside tests directory.
+For the deployment of any other contract, just update contract name from 'Storage' to desired contract and provide constructor arguments accordingly 
+in the file `deploy_with_ethers.ts` or  `deploy_with_web3.ts`
+
+Also, there is a script containing some Mocha-Chai unit tests for 'Storage' contract inside 'tests' directory.
```
Comment (ryestew) [inline libs/remix-ws-templates/src/templates/remixDefault/README.txt:21]: In the 'tests' folder there is a script containing Mocha-Chai unit tests for 'Storage' contract.

File: libs/remix-ws-templates/src/templates/remixDefault/README.txt [hunk 4]
```diff
@@ -2,25 +2,27 @@ REMIX DEFAULT WORKSPACE
...
 
 To run a script, right click on file name in the file explorer and click 'Run'. Remember, Solidity file must already be compiled.
 Output from script will appear in remix terminal.
 
-Please note, 'require' statement is supported in a limited manner for Remix supported modules.
-For now, modules supported by Remix are ethers, web3, swarmgw, chai, remix and hardhat only for hardhat.ethers object/plugin.
-For unsupported modules, an error like this will be thrown: '<module_name> module require is not supported by Remix IDE will be shown.'
\ No newline at end of file
+Please note, require/import is supported in a limited manner for Remix supported modules.
+For now, modules supported by Remix are ethers, web3, swarmgw, chai, multihashes, remix and hardhat only for hardhat.ethers object/plugin.
```
Comment (ryestew) [inline libs/remix-ws-templates/src/templates/remixDefault/README.txt:27]: I'm not sure that swarm still supports swarmgw - but lets leave it for now.

**Review by ryestew — CHANGES_REQUESTED** [review]

File: libs/remix-ws-templates/src/templates/remixDefault/README.txt (hunk 4 above)
Comment (Aniket-Engg) [reply libs/remix-ws-templates/src/templates/remixDefault/README.txt:27]: ok

File: libs/remix-ws-templates/src/templates/remixDefault/README.txt (hunk 1 above)
Comment (Aniket-Engg) [reply libs/remix-ws-templates/src/templates/remixDefault/README.txt:16]: done

**Review by ryestew — CHANGES_REQUESTED** [review]

**Review by ryestew — APPROVED** [review]

**Pull request merged**